Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ARKit
+import ArcGISToolkit
+import ArcGIS
+
+open class ARExample: UIViewController {
+    
+    public let arView = ArcGISARView(frame: CGRect.zero)
+
+    override open func viewDidLoad() {
+        super.viewDidLoad()
+        
+        //
+        // Example of how to get ARSessionDelegate methods from the ArcGISARView
+        //
+        arView.sessionDelegate = self
+
+        view.addSubview(arView)
+        arView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            arView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            arView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            arView.topAnchor.constraint(equalTo: view.topAnchor),
+            arView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+            ])
+        
+        arView.sceneView.scene = scene()
+    }
+    
+    override open func viewDidAppear(_ animated: Bool) {
+        arView.startTracking()
+    }
+    
+    override open func viewDidDisappear(_ animated: Bool) {
+        arView.stopTracking()
+    }
+    
+    private func scene() -> AGSScene {
```

Comment:
you can make this a var (calculated property), or a lazy var (lazy calculated property) if you only ever want it created once.

Reply:
changed to property.